src/index.tsx

```diff
@@ -1,10 +1 @@
-import * as React from 'react';
-
 import './index.scss';
```

Comment:
You will need to add the export statements here to export the components that you've exported in index.ts in the Button directory. This is the root index of SRC